### DISCUSSION TITLE
Bump included mkdocs-git-committers-plugin-2 version

### DISCUSSION THREAD

**Discussion by saltydk** [squidfunk/mkdocs-material#7840] [Q&A] [answered]
Wondering if there is a particular reason it is set to a fairly old version.

**Comment by squidfunk** [comment]
I haven't had the time to check whether the latest version works. PR appreciated! If you can confirm that everything works as expected, we can bump the version to 2.x. We should keep using semantic versioning.

**Reply by niclasheinz** [reply]
I'll check it today. I'll get back to you

**Reply by niclasheinz** [reply]
I've tested it locally. It works seemlessly #7844

**Reply by niclasheinz** [reply, marked as answer]
@saltydk the PR has been merged, now we are up to date.

**Reply by saltydk** [reply]
Thanks
